distribution/changelog.txt

```diff
@@ -28,6 +29,7 @@
 - Removed: BMP screenshots.
 - Removed: Intamin and Phoenix easter eggs.
 - Fix: [#1038] Guest List is out of order.
+- Fix: [#2081] Infinite loop.
```

Comment:
I have updated the issue to something more meaningful:
`Game hangs when track has infinite loop`